aguskoll/target-app
# Target app: sign up

### **What's in**

- The design of the sign in and the sign up pages.
- The sign up logic with the backend. 
- The sign up form state logic.
- Unit testing 
- App logo

### **How it looks**  

https://drive.google.com/file/d/1OLYh04CpiNt1daSYoBG6TuXvALmiPieD/view?usp=sharing

### **How to test**

1. Run this branch 
2. Go to sign up page
3. Enter the correct data
4. Press sign up
5. Validate the login 
6. Log out 
7. Enter wrong data
8. Press sign up
9. Validate the Errors
10. Enter the same mail as in step 4
11. Check you see the error msg "email has already been taken"
